src/frontend/komponenter/ServiceContext.tsx

```diff
@@ -16,6 +20,9 @@ const [ServiceProvider, useServiceContext] = constate(() => {
     const [taskerTilOppfølging, settTaskerTilOppfølging] = useState<{
         [key: string]: IOppfølgingstask;
     }>({});
+    const [taskerFeiletOgManuellOppfølging, settTaskerFeiletOgManuellOppfølging] = useState<
+        IAntallFeiletOgManuellOppfølging[]
+    >([]);
```

Comment:
Er det nødvendig å legge til denne staten? Jeg ser at `taskerFeiletOgManuellOppfølging` kun blir hentet ut et sted, og jeg tror vi kunne spart oss for en del kode dersom vi hadde unngått å involvere ServiceContext i dette tilfellet.